src/types/authForms/index.ts

```diff
@@ -0,0 +1,8 @@
+export interface SignInSignUpValues {
+  name?: string;
+  email: string;
+  password: string;
+  confirmPassword?: string;
+}
+
+export type FormValues = 'name' | 'email' | 'password' | 'confirmPassword';
```

Comment:
try this, it will be automatically updated if `SignInSignUpValues` will expanded

```suggestion
export type FormValues = keyof SignInSignUpValues;
```